deeprankcore/feature/atom.py

```diff
@@ -0,0 +1,53 @@
+from deeprankcore.models.structure import Atom
+from deeprankcore.models.graph import Graph
+from deeprankcore.models.forcefield.vanderwaals import VanderwaalsParam
+from deeprankcore.domain.forcefield import atomic_forcefield
+from deeprankcore.models.error import UnknownAtomError
+import logging
+
+
+_log = logging.getLogger(__name__)
+
+
+# these will ultimately be moved to deeprankcore.domain.feature.py and imported in this file.
+# for now, I am keeping them here to avoid merge conflicts with the feature renaming branch.
+
+# from models.structure.Atom
+FEATURE_NODE_ATOMICELEMENT = "atomic_element" # AtomicElement object
+FEATURE_NODE_PDBOCCUPANCY = "pdb_occupancy" # float(0 < x < 1)
+
+# from feature.atomic_contact.add_features_for_atoms
+FEATURE_NODE_ATOMICCHARGE = "atomic_charge" # float
+FEATURE_NODE_VANDERWAALS = "vanderwaals" # ?
```

Comment:
These will move to deeprankcore.domain.feature.py.
I am not doing this yet because I am editing that file in another branch and it would probably create a merge conflict.